### PR TITLE
agenda April 19, 2023

### PR DESCRIPTION
**Pull request by andorsk**

### CODE REVIEW

**Comment by TallTed** [comment]
I cannot urge strongly enough that you use the ISO standard date format for things like this, i.e., `YYYY-MM-DD`, instead of the non-standard and often ambiguous `mmdd[yy]yy`, `ddmm[yy]yy`, etc.

File: docs/agenda.md [hunk 1]
```diff
@@ -23,60 +23,79 @@ Bi-weekly DIF call notes on DWN specifications.
 - Andor Kesselman @andorsk email: andor@benri.io
 - Liran Cohen @lirancohen
 
+## DIF Meeting March 19, 2023
```
Comment (TallTed) [inline docs/agenda.md:26]: I'm fairly certain this section was meant to be dated today.
```suggestion
## DIF Meeting 2023-04-19
```

Comment (andorsk) [reply docs/agenda.md:26]: You are correct @TallTed! Thanks!

**Comment by andorsk** [comment]
> I cannot urge strongly enough that you use the ISO standard date format for things like this, i.e., `YYYY-MM-DD`, instead of the non-standard and often ambiguous `mmdd[yy]yy`, `ddmm[yy]yy`, etc.

@TallTed fair point about the date format. Will do moving forward.

**Pull request merged**